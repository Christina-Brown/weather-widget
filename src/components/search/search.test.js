import React from "react";
import { shallow } from "enzyme";
import Search from "./search";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Search />);
});

it("should render search without crashing", () => {
  expect(wrapper).toMatchSnapshot();
});
