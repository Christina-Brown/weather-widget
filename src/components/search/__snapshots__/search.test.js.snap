// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render search without crashing 1`] = `
<div
  className="search"
>
  <input
    className="search-bar"
    placeholder="Search a city..."
    type="text"
  />
</div>
`;
